common/djangoapps/entitlements/tests/test_models.py

```diff
@@ -0,0 +1,105 @@
+"""Test Entitlements models"""
+
+import unittest
+from datetime import datetime, timedelta
+
+import pytz
+from django.conf import settings
+from django.test import TestCase
+
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+
+# Entitlements is not in CMS' INSTALLED_APPS so these imports will error during test collection
+if settings.ROOT_URLCONF == 'lms.urls':
+    from entitlements.tests.factories import CourseEntitlementFactory, CourseEntitlementPolicyFactory
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class TestModels(TestCase):
+    """Test entitlement with policy model functions."""
+
+    def setUp(self):
+        super(TestModels, self).setUp()
+        self.course = CourseOverviewFactory.create(
+            start=datetime.utcnow()
+        )
+        self.user = UserFactory(is_staff=True)
+        self.enrollment = CourseEnrollmentFactory.create(user=self.user, course_id=self.course.id)
```

Comment:
Unless you need the self.user and self.course.id consider using the ones that are created by default when just using 

`CourseEnrollmentFactory()`

Without the .create(...)

Reply:
I did need the course.id I can remove the user though.